### PR TITLE
moved mobile integ tests to ci nightly

### PR DESCRIPTION
**Pull request by Oscar-Pepper**

### CODE REVIEW

File: .github/workflows/ci-nightly.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      cache-key: ${{ needs.create-cache-key.outputs.cache-key }}
+      arch: armeabi-v7a
+
+  integration-test-android:
+    uses: zingolabs/zingo-mobile/.github/workflows/integration-test.yaml@dev
+    needs: [ create-timestamp, create-cache-key, build-android ]
+    with:
+      timestamp: ${{ needs.create-timestamp.outputs.timestamp }}
+      cache-key: ${{ needs.create-cache-key.outputs.cache-key }}
+      abi: armeabi-v7a
```
Comment (juanky201271) [inline .github/workflows/ci-nightly.yaml:28]: in CI nightly only for one arch? I don't know if can be a good idea to do for the 4 archs...????

Comment (Oscar-Pepper) [reply .github/workflows/ci-nightly.yaml:28]: i can add all four archs good idea

**Review by juanky201271 — DISMISSED** [review]
IMO, looks good...

**Review by juanky201271 — APPROVED** [review]
IMO still looks good.

**Pull request merged**